app/views/home/_footer_large.html.erb

```diff
@@ -50,7 +50,7 @@
         "email": '<%= current_user.present? ? current_user.email : "" %>',
         "name": '<%= current_user.present? ? current_user.name : "" %>',
         "properties": {
-                "Club": '<%= current_user.present? ? current_user.organisation : "" %>'
```

Comment:
This will work, but looks slightly odd. There is a new method called organisation in the User model which returns a string of the club name, however, there may be instances of organisation as legacy code that could require an object, rather than a string of the club name.

Reply:
This is to handle the case where the user hasn't signed in (ie anonymous browsing), and we don't want the page to crash. If they are logged in, then we populate the Doorbell widget. Is there a better way?

The organisation method was updated to return the name only, due to this refactoring. This was for backwards compatibility. Testing has not surfaced any legacy code that breaks with this change (yet).

Reply:
Happy to push it out and we can refactor later if needed.